data-serving/data-service/src/model/case-reference.ts

```diff
@@ -31,6 +32,9 @@ export type CaseReferenceDocument = mongoose.Document & {
     /** The URL of the source of the case data at the time of ingestion. */
     sourceUrl: string;
 
+    /** The UUID of the upload by which this case was entered into the DB. */
```

Comment:
maybe make some reference to a batch of cases? upload to me could be single, batch, etc

Reply:
Sounds good. I'll clarify with a description in the OpenAPI spec as well.